src/plugins/raven/selectors/getCollaborationNodes.js

```diff
@@ -0,0 +1,3 @@
+import getNode from '@triniti/cms/plugins/ncr/selectors/getNode';
+
+export default (state) => Object.keys(state.raven.collaboration).map((nodeRef) => (nodeRef === 'acme:article:fake-article' || nodeRef === 'general' ? null : getNode(state, nodeRef))).filter((node) => !!node);
```

Comment:
@nanetteCodes lets make constants for both `'acme:article:fake-article'` and `'general'`

Reply:
added constants